data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -130,8 +130,9 @@
                                 description= _ "This massive blade was created centuries ago by long-forgotten elvish forgemasters, who imbued the bluish steel with an inner magical fire. Tongues of flame dance on the surface, giving the metal a flawless mirrored finish."
                                 cannot_use_message= _ "Only the leader of an army can wield this sword!"
                                 [filter]
-                                    type=Fighter,Commander,Lord,Princess,Battle Princess,Elvish Captain,Elvish Hero,Elvish Marshal,Elvish Champion,Paladin,Elvish Lord,Elvish High Lord
+                                    type_adv_tree=Fighter,Princess,Elvish Captain,Elvish Hero,Paladin,Elvish Lord
                                     x,y={X},{Y}
+                                    side=1
```

Comment:
ok to add but doesnt do anything

Reply:
Might be to satisfy the schema validator?

Reply:
removed